src/utils/scroll.js

```diff
@@ -0,0 +1,179 @@
+
```

Comment:
are other things same ? except using e.originalEvent

Reply:
Yep, they're mostly the same.